syntax = "proto3";

package jio.brain.proto.knowledge.api.data;

option java_package = "com.jio.brain.proto.knowledge.api.data";
option java_multiple_files = true;

import "jio/brain/proto/base/status.proto";
import "jio/brain/proto/knowledge/base/biz_id.proto";

service UpdateBizidToEntityService {
  rpc serve(UpdateBizidToEntityRequest) returns (UpdateBizidToEntityResponse) {}
}

message UpdateBizidToEntityRequest {
  string entity_id = 1;
  jio.brain.proto.knowledge.base.BizId old_biz_id = 2;
  jio.brain.proto.knowledge.base.BizId new_biz_id = 3;
}

message UpdateBizidToEntityResponse {
  jio.brain.proto.base.BrainStatus status = 1;
}
